src/EA.Weee.Web/Areas/Admin/Views/Aatf/Details.cshtml

```diff
@@ -274,7 +274,7 @@
                 @if (Model.Organisation.OrganisationType == EA.Weee.Core.Organisations.OrganisationType.RegisteredCompany)
                 {
                     <tr class="govuk-table__row">
-                        <th class="govuk-table__header" scope="row">Company registration number (CRN)</th>
+                        <th class="govuk-table__header" scope="row">company registration number (CRN)</th>
```

Comment:
Dont think this should change as its heading.

Reply:
restored